### ISSUE TITLE
Histogram plot isn't working anymore

### ISSUE THREAD

**Issue by AdrianAntico** [JohnCoene/echarts4r#559]
For some reason the histogram is no longer generating an output plot. Below is code from the examples on the home page...

```r
# data.frame
df <- data.frame(
  x = 1:100,
  y = rnorm(100, 20, 12)
)

df |> 
  e_charts() |> 
  e_histogram(y) |> 
  e_tooltip()
```

**Comment by munoztd0** [comment]
Yeah you have to add name = "histogram" now

``` r
library(echarts4r)

df <- data.frame(
  x = 1:10,
  y = rnorm(10, 20, 12)
)

plot = df |> 
  e_charts() |> 
  e_histogram(y, name = "histogram") |> 
  e_tooltip()

plot
```

![](https://i.imgur.com/w1uwHNo.png)

![image](https://github.com/JohnCoene/echarts4r/assets/43644805/1464c842-4aab-4c2e-979e-6c13e138741a)

I am going to change the doc or make it by default so it work with the legacy code.

thanks for noticing us !

**Comment by AdrianAntico** [comment]
@munoztd0 thanks for the quick response. This is a great package that I utilize often

**Comment by AdrianAntico** [comment]
@munoztd0 I ran into something similar with the bar plot using the polar option. Is there a similar update to that plot call as well?

```r
df <- data.frame(x = 1:10, y = seq(1, 20, by = 2))

df |>
  e_charts(x) |>
  e_polar() |>
  e_angle_axis() |>
  e_radius_axis() |>
  e_line(y, coord.system = "polar", smooth = TRUE)
```

**Comment by munoztd0** [comment]
``` r
library(echarts4r)

df <- data.frame(x = 1:10, y = seq(1, 20, by = 2))
 
plot <- df |>
  e_charts(x) |>
  e_polar() |>
  e_angle_axis() |>
  e_radius_axis() |>
  e_line(y, coord_system = "polar", smooth = TRUE)

plot
```
![image](https://github.com/JohnCoene/echarts4r/assets/43644805/fa0077f3-7c8f-4ce5-93a1-a67f9c522503)


<sup>Created on 2023-12-21 with [reprex v2.0.2.9000](https://reprex.tidyverse.org)</sup>

So it's just that coord.system is deprecated by coord_system apparently, I'm gonna change it now, thanks for noticing.

**Comment by munoztd0** [comment]
fixed in #606
